### PR TITLE
Feature/block disjunction

### PR DESCRIPTION
**Pull request by gnusi**

### CODE REVIEW

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/iresearch_s-pr/118/

File: core/utils/levenshtein_utils.cpp [hunk 1]
```diff
@@ -133,7 +133,7 @@ class parametric_state {
         if (subsumes(new_pos, *begin)) {
           std::swap(*begin, positions_.back());
```
Comment (Dronplane) [inline core/utils/levenshtein_utils.cpp:134]: Use swap_remove here?

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/iresearch_s-pr/122/

**Comment by gnusi** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/iresearch_s-pr/124/

**Comment by gnusi** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/iresearch_s-pr/125/

**Pull request merged**